### ISSUE TITLE
gist:ContemporaneousEvent incorrect rdfs:comment

### ISSUE THREAD

**Issue by marksem** [semanticarts/gist#174]
Comments says: "An event that actually started after the present time.  When we record an end time it ceases to be contemporaneous"

Isn't this incorrect to say it started _after_?  Isn't this supposed to be that it started _before_ the current time, and is still ongoing?

**Comment by uscholdm** [comment]
Yeah, I think so. Good catch.

**Comment by marksem** [comment]
@rjyounes , bringing this back to our awareness.  Fix should just be changing comment to something like:
"An event that actually started **before** the present time.  When we record an end time it ceases to be contemporaneous."

**Comment by rjyounes** [comment]
I've put this on the June project to get it back on  our radar.

Comments:
- I recommend changing the name altogether. Contemporaneous refers to TWO times that occur at the same time.  I have not seen any dictionary that defines it as contemporaneous _with the current time_ without explicitly specifying that. How about CurrentEvent? 
- The contemporaneous event could have started before _or at_ the current moment in time. (I guess this is a quibble since as soon as we've recorded that start time it's already in the past.)
- The contemporaneous event ceases to be contemporaneous only if the recorded end time is _before_ the current time. If it is a future time we would still refer to it as contemporaneous.

I've added some additional questions related to our `Event` class hierarchy in #489. We may want to consider these in tandem, or we may want to address this smaller issue first and the larger issues in #489 later.
